src/privileges.js

```diff
@@ -8,6 +8,9 @@ privileges.userPrivilegeList = [
 	'topics:read',
 	'topics:create',
 	'topics:reply',
+	'edit',
+	'delete',
```

Comment:
Should we call these `posts:edit` and `posts:delete` since we use `topics:*` for topic related privs.